src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,29 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+type Todos = {
+  todo: Todo;
+};
```

Comment:
Use conventional name for this type.
```suggestion
type Props = {
  todo: Todo;
};
```